dimagi/pages/templates/base.html

```diff
@@ -223,7 +223,20 @@ <h3>
       {% require_module 'app' %}
     {% endif %}
 
-    {% block js %}{% endblock %}
-
+    {% block js %}
+      <!--[if lte IE 8]>
```

Comment:
hmmm...no this is definitely not my intention with the "make subscription form global" comment. You need to add it to `app.js` just like `demoForm`. Also, please never add anything in the `{% block js %}` of the base template.